internal/printer/print-to-json.go

```diff
@@ -204,11 +204,17 @@ func renderNode(p *printer, parent *ASTNode, n *Node, opts t.ParseOptions) {
 			}
 
 			for _, attr := range n.Attr {
+				var name string
+				if attr.Namespace != "" {
+					name = attr.Namespace + ":" + attr.Key
+				} else {
+					name = attr.Key
+				}
```

Comment:
Definitely nothing wrong with your approach but I would find this a little bit cleaner!

- Assign to `name` only if `attr.Namespace != ""`
- Use `fmt.Sprintf()` to concat namespace and key.

```suggestion
				name := attr.Key
				if attr.Namespace != "" {
					name = fmt.Sprintf("%s:%s", attr.Namespace, attr.Key)
				}
```